### PR TITLE
Feat/vote bottom

### LINKED ISSUE #197: [feat] Disable vote btn when wallet not connected
**Is your feature request related to a problem? Please describe.**
Currently we are letting the user use the vote btn even if he is not connected.

**Describe the solution you'd like**
Please disable the btn and display a message indicating the user to connect to his wallet if he wants to vote for a funding.

### PR DESCRIPTION
**Pull request by josephpdf**
# Pull Request

- [x] Closes #197 
- [x] Added tests (if necessary)
- [x] Run tests
- [x] Run formatting
- [x] Commented the code

## Changes description

I modified the button behavior based on the wallet connection status:
- When no wallet is connected, the vote button remains disabled and added a short description.
- Upon connecting a wallet, the button becomes active and clickable.

## Current output
- **Disconnected wallet:** Button is disabled.
Captura de pantalla 2024-11-07 120943
- **Connected wallet:** Button is enabled.
Captura de pantalla 2024-11-07 120923

## Time spent breakdown

1. **Research & Planning:** 1 hour
2. **Code & Testing:** 1.5 hour
3. **Code review and formatting:** 30 minutes

## Comments

This update improves the user experience by ensuring that the vote button only becomes interactive when a wallet is connected, aligning with the intended app functionality.

### CODE REVIEW

File: frontend/gostarkme-web/next.config.mjs
Comment (adrianvrj) [inline frontend/gostarkme-web/next.config.mjs:1]: please do not upload the changes to this file, revert

File: frontend/gostarkme-web/.env.example
Comment (adrianvrj) [inline frontend/gostarkme-web/.env.example:1]: please revert the changes to this file

File: frontend/gostarkme-web/components/modules/Fund/FundVote.tsx [hunk 1]
```diff
@@ -38,7 +38,21 @@ export const FundVote = ({ upVotes, upVotesNeeded, addr, setLoading, getDetails
...
+      ) : ( // If the wallet is not connected, render a disabled vote button with instructions
+        <div className="text-center"> 
+          <Button 
+            label="Vote" 
+            onClick={() => {}} 
+            className="opacity-50 cursor-not-allowed"
+            disabled={true}
+          />
+          <p className="text-sm text-gray-500 mt-2">
+          Connect your wallet to be able to vote
```
Comment (adrianvrj) [inline frontend/gostarkme-web/components/modules/Fund/FundVote.tsx:52]: could you change this for "Connect your wallet to vote" please

**Review by adrianvrj — CHANGES_REQUESTED** [review]
just a few changes

File: frontend/gostarkme-web/.env.example
Comment (josephpdf) [reply frontend/gostarkme-web/.env.example:1]: I didn't know that the change was allowed, both files are now as before

File: frontend/gostarkme-web/components/modules/Fund/FundVote.tsx (hunk 1 above)
Comment (josephpdf) [reply frontend/gostarkme-web/components/modules/Fund/FundVote.tsx]: Change made
